common/source/docs/common-blheli32-passthru.rst

```diff
@@ -1,62 +1,158 @@
 .. _common-blheli32-passthru:
 
-BLHeli32/ BLHeli-S Pass-Through Support
-=======================================
+BLHeli32 and BLHeli_S ESCs
+==========================
 
-In addition to offering enhanced capabilities such as reversing and often, telemetry, BLHeli capable ESCs have the ability to have their parameters and features programmed through the autopilot while it is connected to a PC (if the autopilot firmware provides this capability also). This BLHeli pass-through protocol allows you to configure and upgrade your ESCs without having to disconnect them from your vehicle. You can plug a USB cable into your autopilot and run the BLHeliSuite or BLHeliSuite32 software for Windows to configure your ESCs. 
+The BLHeli firmware and configuration applications were developed to allow the configuration of ESCs and provide additional features. ESCs with this firmware allow configuring timing, motor direction, LEDs, motor drive frequency, etc.  Before attempting to use BLHeli please follow the :ref:`DShot setup instructions <common-dshot-escs>`.
 
-Most BLHeli ESCs can autodetect if PWM or DShot is selected by the motor protocol parameter set by the user. See the configuration section of :ref:`common-brushless-escs` for more information. In addition, some BLHeli32 capable ESCs offer bi-directional capabilities for telemetry reporting over the control link, see the section below.
+This page includes setup instructions for the following features
 
-.. note::
-   ArduPilot firmware supports the pass-through protocol with up-to-date BLHeli_32 firmware and BLHeliSuite32, or BLHeli_S firmware and BLHeliSuite only.
+- Pass-Through support allows the BLHeli application to be used to configure the ESCs while remaining connected to the autopilot
+- :ref:`Reversible DShot <blheli32-reversible-dshot>` (aka 3D mode) allows the motor to be spun in either direction
+- :ref:`Bi-directional Dshot <bidir-dshot>` allows the ESCs to send RPM back to the autopilot without the need for an additional telemetry connection
+- :ref:`ESC Telemetry <blheli32-esc-telemetry>` allowing the ESCs to send RPM, voltage and current information back to the autopilot so that it can be logged, viewed in real-time or even allow the removal of a :ref:`battery monitor <esc-telemetry-based-battery-monitor>`
```

Comment:
should be "allows the" to be consistent with the other bullets

Reply:
changed to "allows" txs.